scripts/pr_review_test.py

```
@@ -11,6 +11,11 @@
 import unittest
 from unittest import mock
 
+try:
+    import yaml
+except ImportError as exc:  # pragma: no cover
+    raise RuntimeError("PyYAML is required to verify PR-review workflow structure") from exc
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)